src/screens/ChannelListScreen/index.tsx

```diff
@@ -53,12 +71,26 @@ const ChannelListScreen = () => {
   };
 
   return (
-    <FlatList
-      data={channels}
-      keyExtractor={(item) => item.id}
-      scrollEnabled={false}
-      renderItem={renderChannelItem}
-    />
+    <>
+      <StatusBar translucent={false} />
+      <View style={{height: 52}}>
+        <Search route={route} onPress={goToContactScreen} />
+      </View>
+
+      <FlatList
+        data={channels}
+        keyExtractor={(item) => item.id}
+        ListHeaderComponent={
+          <ChannelListHeaderItem
+            name={`@${profile?.username}`}
+            picture={profile?.profile_pic_path}
+            type="SIGNED"
+            testID="horizontal-tab-0"
+          />
+        }
+        renderItem={renderChannelItem}
+      />
+    </>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `StatusBar` and `Search` component at the top of the screen, and wraps the `FlatList` in a fragment. The `FlatList` now includes a `ListHeaderComponent`, which is a `ChannelListHeaderItem`. 

It's worth noting that the `keyExtractor` prop of `FlatList` assumes that `item.id` is unique. If there's a chance that two items could have the same ID, this could lead to rendering issues. Make sure that `item.id` is always unique.

Also, the `renderItem` prop of `FlatList` is set to `renderChannelItem`, but `renderChannelItem` is not defined in the provided code. Make sure that this function is defined and correctly renders each item.

Finally, there's no error boundary around the `FlatList`. If an error occurs while rendering an item (for example, if `renderChannelItem` throws an error), it will cause the entire component to unmount. Consider adding an error boundary to handle any errors that occur during rendering.

```diff
      <FlatList
        data={channels}
        keyExtractor={(item) => item.id}
        ListHeaderComponent={
          <ChannelListHeaderItem
            name={`@${profile?.username}`}
            picture={profile?.profile_pic_path}
            type="SIGNED"
            testID="horizontal-tab-0"
          />
        }
        renderItem={renderChannelItem}
      />
+        </ErrorBoundary>
    </>
```
In the above code, `ErrorBoundary` is a component that catches JavaScript errors anywhere in their child component tree, logs those errors, and displays a fallback UI.


<!-- This is an auto-generated comment by OSS CodeRabbit -->